scylla/src/transport/session_builder.rs

```diff
@@ -29,6 +29,10 @@ use openssl::ssl::SslContext;
 use tracing::warn;
 
 mod sealed {
+    // This is a sealed trait - its whole purpose is to be unnameable.
+    // This means we need to disable the check.
+    #[allow(unknown_lints)] // Rust 1.66 doesn't know this lint
```

Comment:
We no longer support Rust 1.66. Does 1.70 know this lint?

Reply:
Unfortunately not - it was introduced in 1.79, so we would need to bump MSRV much higher for that.